src/components/Field/Select.js

```diff
@@ -0,0 +1,27 @@
+import { SingleSelectField, SingleSelectOption } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const Select = ({ options, selected, onChange, label, ...props }) => (
+    <SingleSelectField
+        onChange={onChange}
+        selected={selected}
+        label={label}
+        {...props}
+    >
+        {options.map((option) => (
```

Comment:
Are there use cases where the dropdowns will have a lot of options? If so, perhaps adding a filter input for the available options is something you might want to consider. Similar to this [DHIS2/ui dropdown example ](https://ui.dhis2.nu/demo/?path=/story/multi-select--with-filter-field).

Reply:
Good idea, I'll add it to some of them, like "programs" and "stockCorrected"